tests/qoi_tests.cpp

```diff
@@ -23,6 +23,12 @@ TEST_CASE("Header : get_header", "[header]") {
   REQUIRE(head.colorspace == 0);  
 }
 
+TEST_CASE("Header : is_valid", "[header]") {
+  std::vector<unsigned char> test_header{0x71, 0x6f, 0x69, 0x65, 0x00, 0x00, 0x03, 0x00, 0x0, 0x0, 0x2, 0x0, 0x3, 0x0, 0xfe, 0x74};
```

Comment:
Test passes fine, but when I try to get the hex in an image and then run them through the C implementation it doesn't seem to be working...

Reply:
The reference implementation should return NULL for this byte sequence. Did it decode it?

Reply:
Nope, I just shouldn't do code reviews late at night :laughing: